queue_processors/autoreduction_processor/post_process_admin.py

```diff
@@ -177,6 +178,13 @@ def _load_reduction_script(self, instrument_name):
         """ Returns the path of the reduction script for an instrument. """
         return os.path.join(self._reduction_script_location(instrument_name), 'reduce.py')
 
+    def reduction_started(self):
```

Comment:
I guess the name comes from the reduction status. But I think something like `send_reduction_started_status()` is more descriptive and less confusing to a new developer of the project.

Reply:
Though this is't code i have changed as part of this PR (it's just a result of a merge conflict resolution), I agree, will change this to be more verbose